example_bytecode_test.go

```diff
@@ -1,4 +1,4 @@
-package duktape_test
+package duktape
```

Comment:
@olebedev we can leave the test we have already as is, we don't need to change that. Just the example, I can make the change right now. I don't thinks we need to do anything with the readme.